.gitignore

```diff
@@ -66,3 +66,4 @@ src/mixxx.rc.include
 src/mixxx.res
 src/test/**/*.actual
 res/qrc_mixxx.cc
+.vscode/settings.json
```

Comment:
This should go to to your private git ignore file like described at the top of this file.